assets/js/base/context/cart-checkout/checkout/processor/index.js

```diff
@@ -47,12 +47,12 @@ const preparePaymentData = ( paymentData ) => {
 const CheckoutProcessor = () => {
 	const {
 		hasError: checkoutHasError,
-		onCheckoutProcessing,
-		onCheckoutCompleteSuccess,
+		onCheckoutBeforeProcessing,
 		dispatchActions,
 		redirectUrl,
 		isProcessing: checkoutIsProcessing,
-		isProcessingComplete: checkoutIsProcessingComplete,
+		isBeforeProcessing: checkoutIsBeforeProcessing,
```

Comment:
I don't think we need to rename these anymore? I don't see any conflicts.

Reply:
True. I think the aliases make it a bit clearer in the implemented code though (so less to avoid conflicts, more to make things readable where implementation occurs - i.e. _what_ is processing?).  Eventually, this could be refactored so logic is extracted into smaller testable bits outside of this component and at that point aliasing might have less value.